charts/linkerd2-multicluster-remote-setup/templates/service-mirror-rbac.yaml

```diff
@@ -14,12 +14,8 @@ metadata:
 rules:
 - apiGroups:
   - ""
-  resources:
-  - services
-  verbs:
-  - list
-  - get
-  - watch
+  resources: ["services", "configmaps"]
```

Comment:
Not sure if we care, but we can scope this permission down further.  We only need permission to read configmaps in the control plane namespace.  Or, even better, we only need permission to read specifically the linkerd config configmap.

Reply:
I looked at that option. I think it would require us to have a separate Role which is namespace scoped only to the control plane namespace. It will not limit us to only the particular config map though. Was not sure if it is worth it. @alpeb wdyt?

Reply:
You can add `subResources` to scope to specific names.